website/content/en/preview/reference/instance-types.md

```diff
@@ -20599,7 +20800,7 @@ below are the resources available with some assumptions and after the instance o
  |--|--|
  |karpenter.k8s.aws/instance-accelerator-count|16|
  |karpenter.k8s.aws/instance-accelerator-manufacturer|aws|
- |karpenter.k8s.aws/instance-accelerator-name|inferentia|
+ |karpenter.k8s.aws/instance-accelerator-name|trainium|
```

Comment:
We should call out in the upgrade guide that trainium instances now have trainium as there accelerator name, rather than inferentia. Same goes for inferentia -> inferentia2 for the inf2 instances. This will be a breaking change for anyone who had label selectors on workloads targeting those instances.